docusaurus.config.js

```diff
@@ -185,7 +152,7 @@ module.exports = {
             },
             {
               label: "Release cycle",
-              to: "docs/reference/release-policy",
+              to: "https://docs.camunda.io/docs/reference/release-policy",
```

Comment:
These are footer links. A few of these pages don't exist in the 0.25 site, so I pointed them at production instead.